var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -333,6 +333,9 @@ class QuantumEspresso(CMakePackage, Package):
     patch_checksum = "72564c168231dd4a1279a74e76919af701d47cee9a851db6e205753004fe9bb5"
     patch(patch_url, sha256=patch_checksum, when="@6.7+qmcpack")
 
+    # 6.7
+    patch("configure.patch", when="@6.6:")
```

Comment:
Please use `https://gitlab.com/QEF/q-e/-/commit/081409ea90cba0ddc07bea5ac29e3cd422c67d3d.diff` patch and also fix the comment `6.7` -> `6.6`

Reply:
LGTM